lib/bademagic_module/bluetooth/ble_state_interface.dart

```diff
@@ -0,0 +1,7 @@
+abstract class BleState {
+  Future<BleState?> isSuccess(String message);
+
+  Future<BleState?> isFailed(String message);
+
```

Comment:
These method can be improvised or removed. Showing Success or Failed Toast can be a Completed State. That state shows a toast